front/src/applications/operationalStudies/views/v2/ScenarioV2.tsx

```diff
@@ -147,6 +179,15 @@ const ScenarioV2 = () => {
     []
   );
 
+  const onClickMicroMacroButton = (isMacroButton: boolean) => {
```

Comment:
- Maybe toggleMicroMacroButton is a better name
- isMacroButton -> isMacroMode
```suggestion
  const toggleMicroMacroButton = (isMacroMode: boolean) => {
```

Reply:
Done !!